packages/modules/vehicles/mercedeseq/api.py

```diff
@@ -0,0 +1,209 @@
+#!/usr/bin/python3
+
+import requests
+import json
+import time
+import logging
+# from datetime import datetime, timezone
+from requests.exceptions import Timeout, RequestException
+from json import JSONDecodeError
+from typing import Tuple
+from modules.vehicles.mercedeseq.config import MercedesEQSoc
+# from paho.mqtt import subscribe as subscribe
+from paho.mqtt import publish as publish
+
+ramdiskdir = '/var/www/html/openWB/ramdisk/'
+moduledir = '/var/www/html/openWB/packages/modules/vehicles/mercedeseq/'
+
+req_timeout = (30, 30)  # timeout for requests in seconds
+
+vehicle = None
+Debug = 2
+
+tok_url = "https://ssoalpha.dvb.corpinter.net/v1/token"
+soc_url_pre_vin = "https://api.mercedes-benz.com/vehicledata/v2/vehicles/"
+soc_url_post_vin = "/containers/electricvehicle"
+
+log = logging.getLogger("soc."+__name__)
+
+soc = None
+range = None
+timestamp = None
+
+
+def socDebugLog(message):
+    log.debug(": " + message)
+
+
+def handleResponse(what, status_code, text):
+    if status_code == 204:
+        # this is not an error code. Nothing to fetch so nothing to update
+        socDebugLog(what + " Request Code: " + str(status_code) +
+                    " (no data is available for the resource)")
+        socDebugLog(text)
```

Comment:
```suggestion
        log.debug(what + " Request Code: " + str(status_code) +
                    " (no data is available for the resource)")
        log.debug(text)
```
Entsprechend auch an den anderen Stellen.

Reply:
done